src/community/advanced-data-card.css.ts

```diff
@@ -95,6 +103,8 @@ export const cardContentStyle = style([
     },
 ]);
 
+export const zindex = style({zIndex: -20});
```

Comment:
why -20?

Reply:
The negative z-index is so that images in slots (like SimpleBlock) don't overlap the Touchable

Reply:
this number doesn't make sense to me. I'm sure you can reorder the elements, use different stacking contexts or using a more sensible zIndex value than -20